src/utils.js

```diff
@@ -100,6 +100,16 @@ export const protocolFromUrl = function(url: string): string {
     return (protocol != null ? protocol[1] : "_relative");
 };
 
+/**
+ * Round `n` to 4 decimal places, or to the nearest 1/10,000th em. The TeXbook
+ * gives an acceptable rounding error of 100sp (which would be the nearest
+ * 1/6551.6em with our ptPerEm = 10):
+ * http://www.ctex.org/documents/shredder/src/texbook.pdf#page=69
+ */
+const round = function(n: number): number {
+    return +n.toFixed(4);
+};
+
```

Comment:
I wonder why this file fails to be syntax-highlighted.

Reply:
Dunno. It's syntax highlighted when I open it in VS Code, and it's syntax highlighted when I hit the "Edit file" menu selection here on GitHub. But not these two lines in the viewer. Strange.